### PR TITLE
Restructure stories folder

### PR DESCRIPTION
**Pull request by libertymayc**
- Moves all stories to component named subfolder under /stories folder
- Moves any story assets in /stories/assets
- Moves any story components into /stories/components

### CODE REVIEW

**Review by joshwooding — CHANGES_REQUESTED** [review]
Needs linting

File: packages/lab/stories/components/CloseTabWarningDialog.tsx [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import { Button } from "@salt-ds/core";
 import {
-  TabDescriptor,
   Dialog,
   DialogTitle,
   DialogContent,
   DialogActions,
+  TabDescriptor,
```
Comment (joshwooding) [inline packages/lab/stories/components/CloseTabWarningDialog.tsx:7]: Are the changes in this file necessary? They don't look to be restructuring?

File: packages/lab/stories/tabs.stories.tsx [hunk 2]
```diff
@@ -151,11 +151,11 @@ export const Controlled: TabsStory = () => {
     undefined
   );
   const [tabs, setTabs] = useState([
-    "Home",
-    "Transactions",
-    "Loans",
-    "Checks",
-    "Liquidity",
+    { label: "Home", closeable: false },
```
Comment (joshwooding) [inline packages/lab/stories/tabs.stories.tsx:154]: Are the changes in this file necessary? They don't look to be restructuring?

File: packages/lab/stories/components/CloseTabWarningDialog.tsx (hunk 1 above)
Comment (libertymayc) [reply packages/lab/stories/components/CloseTabWarningDialog.tsx:7]: this CloseTabWarningDialog was used by 2 files where one was wanting the tabs to be string and the other a TabDescriptor, so i merged the types when i moved it to /components

File: packages/lab/stories/tabs.stories.tsx (hunk 2 above)
Comment (libertymayc) [reply packages/lab/stories/tabs.stories.tsx:154]: (+ my other comment) here i just copied the tabs from tabstrip.stories to have the TabDescriptor type

**Comment by joshwooding** [comment]
@libertymayc Seems like there are some conflicts to resolve

**Review by joshwooding — APPROVED** [review]

**Pull request merged**